packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.ts

```diff
@@ -254,9 +254,11 @@ function serializeNode(
       if (shouldBeHidden) {
         const { width, height } = (n as HTMLElement).getBoundingClientRect()
         attributes = {
+          id: attributes.id,
```

Comment:
Is the `id` attribute needed?

Reply:
See my PR description:

> Additionally, we set the ID as a/ it helps with testing, b/ we can
assume it doesn't contain PII and c/ it may be used when
rebuilding (CSS rules could be selecting by ID).

Not strictly needed, but I figured I wouldn't harm?

Reply:
Ok it looks fine